velox/common/memory/HashStringAllocator.cpp

```diff
@@ -232,12 +262,39 @@ void HashStringAllocator::freeRestOfBlock(Header* header, int32_t keepBytes) {
   free(newHeader);
 }
 
+//  static
+int32_t HashStringAllocator::freeListSizes_[kNumFreeLists + 1] = {
+    72,
+    8 * 16 + 20,
```

Comment:
nit: It would be nice to add a comment to explain +20. I assume this is alignment padding from AlignedStlAllocator<16>.